### ISSUE TITLE
@keyframes don't work

### ISSUE THREAD

**Issue by radi-cho** [cssinjs/aphrodite-jss#20]
Hello! I tried to add simple keyframes animation to my styles with `aphrodite-jss`.
I tried this:
```
StyleSheet.create((() => {
  ... ... ...
  '@keyframes anim': {
    from: {opacity: 0},
    to: {opacity: 1}
  },
  animationName: 'anim',
  animationDuration: '4s'
  ...
```
Also:
```
animation: anim 1s ease-out;
```
But it didn't work.
I'm using Chrome 60 and it doesn't need `webkit` prefix, for example.

### Am I wrong with the syntax or this is a bug?
_Suggested labels:_ `question`; `example` & if this is bug - `bug`

**Comment by kof** [comment]
Looking into it

**Comment by kof** [comment]
Looks like a bug, care to fix it? I have create a playground out of the current example to try it out. Once it works, I want to sync that example with the repo so that codesandbox example can be updated anytime from the repo. https://codesandbox.io/s/rl2x5wrqvm

**Comment by radi-cho** [comment]
I was confused is it a bug or no and asked you.
Maybe I can't fix it, can you?

**Comment by kof** [comment]
I can, but my priorities will not allow me to do it in the next weeks.

**Comment by radi-cho** [comment]
Oh, next weeks. I'm working on important project and using `ahrodite-jss`. I need keyframes very soon.
Then I'll try to find the problem and fix it.
But i will need your help.

**Comment by radi-cho** [comment]
When I have questions I'll ask you in Direct Message on Twitter. Is that fine?

**Comment by kof** [comment]
JSS has a gitter channel https://github.com/cssinjs/jss , it is the rendering engine behind aphrodite-jss. As long as I don't spend more time answering your questions than to fix it myself it is ok to ask.

**Comment by radi-cho** [comment]
Ok. No problems. I will do everything i can without questions

**Comment by msnider** [comment]
Just left a quick pull that can provide a fix for this: #22

**Comment by kof** [comment]
released in v2.1.0

**Comment by radi-cho** [comment]
Great!
